### ISSUE TITLE
myView.animate() just run 1 time

### ISSUE THREAD

**Issue by ngocdtb93** [MengTo/Spring#148]
my view (autolayout), then i set properties spring for myView, then when i click button, i call myView.animate()
But it just run 1 time, then i click the button again, nothing happen.
Do u have any idea for this issue? Thanks a million times for reading!

**Comment by hirenpanchal1608** [comment]
I am also getting same issue like @ngocdtb93

**Comment by hirenpanchal1608** [comment]
I got the solution for this.

We have to write again layer.animation = "shake" again with layer.animate()

**Comment by ngocdtb93** [comment]
@hirenpanchal1608 yes, this is the solution, but i still dont know why :) maybe it's a bug of Spring?
